roles/vendor/redis/templates/redis_cluster.conf

```diff
@@ -1,13 +1,13 @@
 port {{ redis_port }}
-dir /var/lib/redis_cluster/redis_{{ redis_port }}
+dir /var/lib/{{redis_dir}}/redis_{{ redis_port }}
 appendonly no
 protected-mode no
-maxmemory {{ redis_maxmemory }}
+maxmemory {{ redis_mem }}
 cluster-enabled yes
 cluster-node-timeout 5000
 cluster-config-file /etc/redis_cluster/redis_{{ redis_port }}/nodes_{{ redis_port }}.conf
-pidfile /var/run/redis_cluster/redis_{{ redis_port }}.pid
-logfile /var/log/redis_cluster/redis_{{ redis_port }}.log
+pidfile /var/run/{{redis_dir}}/redis_{{ redis_port }}.pid
+logfile /var/log/{{redis_dir}}/redis_{{ redis_port }}.log
 loglevel notice
 {% if redis_pass %}
 requirepass {{ redis_pass }}
```

Comment:
This seems to be identical with `redis.conf` except some stuff we could put in variables.